### DISCUSSION TITLE
Possible deadline extention

### DISCUSSION THREAD

**Discussion by DavidSchlipf** [IEAWindTask52/LidarAssistedControl#17] [General]
Dear all,

based on our original [announcement ](https://doi.org/10.5281/zenodo.10931629), the intended deadline was June 30.
However, I think it would be nice to extend it to at least the end of July to give more students the opportunity to participate. 

What do you think?

**Comment by Javier-TRI** [comment]
Hi David,

I believe it would be great for the students to have additional time to explore various control solutions. This extra time would allow them to gain a deeper understanding and potentially discover more effective strategies.

**Comment by DavidSchlipf** [comment]
As discussed in the meeting last Tuesday, End of the summer semester (which is usually August/September) would make most sense!

**Reply by Flixlee** [reply]
Hi David, 

from a student perspective, it is well appreciated if the time period would be extended. 
At the moment everyone is busy with their examination preparation.

**Comment by DavidSchlipf** [comment]
Dear all,
we now fixed the deadline to September 30!
Best regards,
David
